File: DDS_Restaurant_Solution/DDS_Restaurant_Solution/Models/Mesa.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace DDS_Restaurant_Solution.Models
{
    public class Mesa
    {
        [Key]
        public int idMesa { get; set; }

        [MaxLength(2)]
        public string capacidad { get; set; }

        public bool estadoMesa { get; set; }

        public int idSucursal { get; set; }
        [ForeignKey("idSucursal")]
        public Sucursal Sucursal { get; set; }
    }
}

File: DDS_Restaurant_Solution/DDS_Restaurant_VB/Models/Mesa.vb
Imports System.ComponentModel.DataAnnotations.Schema
Imports System.ComponentModel.DataAnnotations
Public Class Mesa
    <Key>
    Public Property idMesa As Integer
    <MaxLength(3)>
    Public Property numMesa As String
    <MaxLength(2)>
    Public Property capacidad As String
    Public Property estadoMesa As Boolean
    Public Property idSucursal As Integer
    <ForeignKey("idSucursal")>
    Public Property Sucursal As Sucursal
End Class
